backend/app/services/platforms/spotify_client.rb

```
@@ -0,0 +1,115 @@
+        platform: 'spotify'
+      }
+    end
+
+    def get_json(url)
+      uri = URI.parse(url)
+      req = Net::HTTP::Get.new(uri)
+      req['Authorization'] = "Bearer #{@connection.access_token}"
+      res = Net::HTTP.start(uri.host, uri.port, use_ssl: true) { |h| h.request(req) }
+      code = res.code.to_i
+      raise Error, "Spotify API error #{code}: #{res.body}" unless code.between?(200, 299)
+      JSON.parse(res.body) rescue {}
```

RuboCop (1.76.1)

[convention] Avoid using `rescue` in its modifier form.

(Style/RescueModifier)